packages/imperative/src/operations/src/Operation.ts

```diff
@@ -475,11 +474,7 @@ export abstract class Operation<T> implements ITaskWithStatus {
         for (let x = 0; x < order.length; x++) {
             this.log.info("Cleaning file: " + this.fileToUndo[x]);
             try {
-                if (fs.statSync(order[x]).isDirectory()) {
-                    removeSync(order[x]);
-                } else {
-                    fs.unlinkSync(order[x]);
-                }
+                fs.rmSync(order[x], {recursive: true, force: true});
```

Comment:
cool that you did extra work to simplify and clean this up! 👍 will try to follow your example in my prs :)